web/beacon-app/src/components/layout/Sidebar/Sidebar.tsx

```diff
@@ -72,6 +81,37 @@ function SideBar({ className }: SidebarProps) {
     invariant(appState?.orgID, 'orgID is not defined');
   }, [appState?.orgID]);
 
+  // Get the version number from the status response without the release level and number.
+  const statusVersion = status?.version.match(/^v?(\d+\.\d+\.\d+)/);
+
+  useEffect(() => {
+    const storedAppVersion = localStorage.getItem('appVersion');
+
+    if (env !== 'development') {
+      if (storedAppVersion && storedAppVersion !== statusVersion?.[0]) {
+        const updateAlertToast = toast.error(
+          <div className="flex items-center">
+            <UpdateAlert />
+          </div>,
+          {
+            // Display toast until the user clicks the update button.
+            duration: 999999999,
```

Comment:
I thought it would be best to display the notification in a way so that it doesn't interrupt users if they're in the middle of doing something like creating a project. However, if they change pages before clicking the update button then they should still see the message.